packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,6 +283,31 @@ class EditableText extends StatefulWidget {
   /// The text style to use for the editable text.
   final TextStyle style;
 
+  /// {@template flutter.widgets.editableText.strutStyle}
+  /// The strut style used for the vertical layout.
+  ///
+  /// [StrutStyle] is used to establish a predictable vertical layout.
+  /// Since fonts may vary depending on user input and due to font
+  /// fallback, [StrutStyle.forceStrutHeight] is enabled by default
+  /// to lock all lines to the height of the base [TextStyle], provided by
+  /// [style]. This ensures the typed text fits within the alotted space.
+  ///
+  /// If null, the strut used will is inherit values from the [style] and will
+  /// have [StrutStyle.forceStrutHeight] set to true. When no [style] is
+  /// passed, the theme's [TextStyle] will be used to generate [strutStyle]
+  /// instead.
+  ///
+  /// To disable strut-based vertical alignment and allow dynamic vertical
+  /// layout based on the glyphs typed, use [StrutStyle.disabled].
+  /// {@endtemplate}
```

Comment:
Would you mind adding a URL reference to something that explains the concept of a strut in general?  It's an odd name for those that are not already familiar with it.

Reply:
I can put a link to https://www.w3.org/TR/CSS2/visudet.html#line-height in the docs for `StrutStyle`. Sticking the link everywhere strut is used might cause a bit too much clutter.

Reply:
I was just recommending that it be added to this template.  The other places where this template appears is likely to be the first and only interaction most developers have with this concept, so it seemed like a good place for an external reference.

Reply:
https://en.wikipedia.org/wiki/Strut_(typesetting) may also be a good more generic albeit brief reference.

Reply:
Ahh my bad, missed that this was the template! That's a good idea.